packages/next/src/server/lib/incremental-cache/fetch-cache.ts

```diff
@@ -245,7 +245,7 @@ export default class FetchCache implements CacheHandler {
           cached.tags ??= []
           for (const tag of tags ?? []) {
             if (!cached.tags.includes(tag)) {
-              cached.tag.push(tag)
+              cached.tags.push(tag)
```

Comment:
It'd be good to update the type on line 236 so it's not any to help catch this going forward

Reply:
@wyattjoh yeah we can used that for line 236. Looking at that now.